pegen/c_generator.py

```diff
@@ -374,18 +376,21 @@ def add_var(self, node, names: List[str]) -> Tuple[str, Optional[str]]:
         call: str
         name, call = self.callmakervisitor.visit(node.item)
         type = None
-        if name and name != 'cut':
-            if name.endswith('_var'):
-                rulename = name[:-4]
-                rule = self.rules.get(rulename)
-                if rule is not None:
-                    if rule.is_loop():
-                        type = 'asdl_seq *'
-                    else:
-                        type = rule.type
-                elif name.startswith('_loop'):
+        if not name:
+            return name, type
+        if name.startswith('cut'):
+            return name, 'int'
+        if name.endswith('_var'):
```

Comment:
This looks like a huge change but is just a dedent (as we are handling 'cut' and no name before).